lib/presentation/extensions/send_file_extension.dart

```diff
@@ -418,6 +427,35 @@ extension SendFileExtension on Room {
     return eventId;
   }
 
+  Future<ImageFileInfo> convertHeicToJpgImage(ImageFileInfo fileInfo) async {
+    final convertedFilePath =
+        StorageDirectoryManager.instance.convertFileExtension(
+      fileInfo.filePath,
+      'jpg',
+    );
+    final newPath = await HeifConverter.convert(
+      fileInfo.filePath,
+      output: convertedFilePath,
+    );
```

Comment:
IMO: Should use `try catch` in this case. Because maybe while converting an error may occur.

Reply:
already, try/catch outside
<img width="520" alt="image" src="https://github.com/user-attachments/assets/23483111-d748-4303-be79-c64b9d8d3224">